miden-tx/tests/integration/scripts/faucet.rs

```diff
@@ -141,11 +142,13 @@ fn faucet_contract_mint_fungible_asset_fails_exceeds_max_supply() {
         .compile_tx_script(tx_script_code, vec![(faucet_pub_key, faucet_keypair_felts)], vec![])
         .unwrap();
 
+    let tx_args = TransactionArgs::new(Some(tx_script), None);
+
     // Execute the transaction and get the witness
-    let executed_transaction =
-        executor.execute_transaction(faucet_account.id(), block_ref, &note_ids, Some(tx_script));
+    let transaction_result =
```

Comment:
I don't think we need to change `executed_transaction` into `transaction_result` here and in other places in this file.